### ISSUE TITLE
Symbols in mapping file depending on language standard

### ISSUE THREAD

**Issue by EugeneZelenko** [include-what-you-use/include-what-you-use#377]
How to handle symbols in mapping file, when symbol location depends on language standard? [std::swap](http://en.cppreference.com/w/cpp/algorithm/swap) location is different in C++98/03 and C++11 or newer.

**Comment by vsapsai** [comment]
And what file do we recommend for `std::swap` now? I don't think there is a mechanism to support different public files for the same symbol. If you have different private files for different language standards, file mapping might work. But I don't think that is an option with libc++.

**Comment by EugeneZelenko** [comment]
Is it possible to include one mapping file form other? So common mapping file could be included from C++11 and C++98 file.

Anyway. it's good idea to load mapping files based on compiler flags (-stdlib, -std).

**Comment by kimgr** [comment]
> Is it possible to include one mapping file form other?

Yes, see https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/IWYUMappings.md#mapping-refs

**Comment by EugeneZelenko** [comment]
type_traits contains std::swap definition.
